try:
    def fun(a,b):
        print(a/b)
    fun(10,5)
except:
    print("There is a division error")
    #fun , try , return , except , finally